src/host-config/proxy.ts

```diff
@@ -138,6 +184,32 @@ export class RedsocksConf {
 	}
 }
 
+function dnsToDnsu2t(
+	dns: unknown,
+): { remote_ip: string; remote_port: number } | null {
+	const dnsu2t = {
+		remote_ip: DEFAULT_REMOTE_IP,
+		remote_port: DEFAULT_REMOTE_PORT,
+	};
+
+	if (!DnsConfigFormat.is(dns)) {
```

Comment:
I think validation should probably be performed in the `index` module with the rest of the validation. The endpoint should return a 400 if the input is wrong

Reply:
Removed this code in favor of input validation in `hostConfig.parse`